.github/workflows/trigger.yml

```diff
@@ -0,0 +1,15 @@
+name: Push Images
```

Comment:
Not sure why the file is called `trigger` instead of `push-images` to match the workflow name.

Reply:
The file was commited by accident, will fix. It's just for testing.